wicket-util/src/main/java/org/apache/wicket/util/crypt/SunJceCrypt.java

```diff
@@ -153,4 +193,17 @@ protected KeySpec createKeySpec()
 	{
 		return new PBEKeySpec(getKey().toCharArray());
 	}
+
+	/**
+	 * Create a random salt.
+	 * 
+	 * @return salt
+	 */
+	public static byte[] randomSalt()
+	{
+		// only 8 bytes long supported
```

Comment:
Please add an explanation `why` only 8 bytes long is supported. I honestly don't know why more is not possible. I guess with the time even you may forget the reason.